src/play/utils/Move.py

```diff
@@ -1,15 +1,111 @@
+"""Class that handles moves!
+
+Supports alternative constructors using `from_*`, so that it can be
+constructed from the sgf format, gtp, and matrix locations.
+Same goes for output, using the `to_*` functions
+"""
 
-class Move:
 
+class Move:
     def __init__(self, col=None, row=None, is_pass=False):
         self.col = col
         self.row = row
         self.is_pass = is_pass
 
-    def get_loc_as_matrix_coords(self):
+    @classmethod
+    def from_sgf(cls, string):
+        """Create instance from string using the SGF standard
+
+        Examples
+        --------
+        >>> print(Move.from_sgf('ef'))
+        (4/5)
+        >>> print(Move.from_sgf('AA'))
+        (0/0)
+        >>> print(Move.from_sgf(''))
+        pass
+        """
+        string = string.lower()
+        if string == '':
+            return cls(is_pass=True)
+        else:
+            col = ord(string[0]) - ord('a')
+            row = ord(string[1]) - ord('a')
+            return cls(col, row)
+
+    @classmethod
```

Comment:
Is this good practice to add? For me, coming from the Java-world, this is totally clear and would need no annotation 🤷‍♂️